### DISCUSSION TITLE
Add xrdp setup in the script.

### DISCUSSION THREAD

**Discussion by mithil404** [Yisus7u7/termux-desktop-xfce#12] [Ideas]
Rdp is better than vnc. I tried setting it up on my own. But it didn't work somehow. If possible, add it in the installation script to have it preconfigured.

**Comment by JesusChapman** [comment]
Once you set it up, but it's not worth it, the xrdp de termux has many bugs, and horrible animations, your resource consumption is very high

**Reply by mithil404** [reply]
I was able to get it started. It stutters a lot and the staircase effect is too much. Its nothing like windows rdp.

**Comment by ghost** [comment]
> Rdp is better than vnc. I tried setting it up on my own. But it didn't work somehow. If possible, add it in the installation script to have it preconfigured.

Xrdp in termux is kinda slow, and very buggy, i'm not gonna do it!
